### DISCUSSION TITLE
How to implement Custom Links In Grid?

### DISCUSSION THREAD

**Discussion by Dev-Lopez02** [serenity-is/Serenity#6555] [Help]
hello, does anyone know how I can implement Custom Links In Grid for the first time?

**Comment by arunkumar12712** [comment]
https://github.com/serenity-is/common-features/blob/master/src/Serenity.Demo.BasicSamples/Serenity.Demo.BasicSamples/Grids/CustomLinksInGrid/CustomLinksInGrid.ts

**Reply by Dev-Lopez02** [reply]
but I still can't see it reflected in the grid

![image](https://user-images.githubusercontent.com/108413888/195201225-0c7e57dd-30ab-4342-93b4-3a12733b4c65.png)
